src/current/cockroachcloud/cloud-api.md

```diff
@@ -669,16 +799,17 @@ The service account associated with the secret key must have the Cluster Adminis
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 curl --request DELETE \
-  --url 'https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/sql-users/{sql_username}'' \
+  --url 'https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/sql-users/{sql_username}' \
```

Comment:
Nit: Poor alignment. I'd align `--url` with `--request`.

Reply:
Leaving for consistency with other examples for now